fishy/helper/hotkey/hotkey_process.py

```diff
@@ -39,12 +40,12 @@ def stop():
 
 class HotKey:
     def __init__(self):
-        self.inq = Queue()
-        self.outq = Queue()
+        self.inq = queue.Queue()
+        self.outq = queue.Queue()
 
         self._hotkeys: Dict[Key, Optional[Callable]] = dict([(k, None) for k in Key])
 
-        self.process = Process(target=process.run, args=(self.inq, self.outq))
+        self.process = Thread(target=process.run, args=(self.inq, self.outq))
```

Comment:
using multithreading for hotkey lagged fishy really bad, thats why we used multiprocessing, we can't afford to use multithreading here...

why make this change though? I dont see any reason why it shouldn't work for linux... if it doesn't, maybe investigating why it doesn't rather than, switching it to multithreading